### ISSUE TITLE
[Suggestion] A Releases section for the repository linking to precompiled builds

### ISSUE THREAD

**Issue by LOuroboros** [kwsch/pk3DS#467]
Title.
https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/managing-releases-in-a-repository

It would be pretty helpful and time saving to be able to get the latest stable or beta build of pk3DS from this repository, like many other projects hosted on GitHub do.
There is a link to an Azure repository for beta or bleeding edge releases, but it doesn't seem to work all too well. Either that, or I'm failing to follow the instructions on the Downloads page of pk3DS at ProjectPokemon.

> click on latest run at the top, then click Artifacts (Published), and download the 'drop' folder

**Comment by kwsch** [comment]
I'm not posting releases for tools that are infrequently updated since they already have CI set up to provide builds.

Clicking on the latest run and downloading the artifacts works fine. The "Downloads" section of ProjectPokemon is not linked to by this repository; only the main thread which contains the download tutorial.

**Comment by LOuroboros** [comment]
> I'm not posting releases for tools that are infrequently updated since they already have CI set up to provide builds.

I suppose that means it's too hard or not worth the effort. Alright then. Thanks for the reply.

> Clicking on the latest run and downloading the artifacts works fine.

I was able to find it at last. Turns out I was looking for the actual word "Artifacts", when in the latest run's page at the Azure Repository that word does not appear. What appears instead is the amount of Artifacts directly.

![Untitled995](https://user-images.githubusercontent.com/4485172/98812443-41a79100-2401-11eb-9f40-56564e81620e.png)

And well, now that I checked the actual post of the tool at ProjectPokemon *(for reference, I was looking at pk3DS' page in PP's Downloads Section which doesn't have a visual hint on how to get the latest build of the tool from the Azure Repository)*, it does contain an image pointing out where exactly the Artifacts section was.

So yeah, I'm sorry for having bothered you ^^
